doc/check-options.md

```diff
@@ -74,6 +75,43 @@ All checks allow these global options:
 | `allowImplicit` | `true`  | Allow the explicit role to match the implicit role of the element |
 | `ignoredTags`   | `[]`    | Do not check for allowed roles in the provided HTML elements list |
 
+### aria-prohibited-attr
+
+<table>
+  <thead>
+    <tr>
+      <th>Option</th>
+      <th align="left">Default</th>
+      <th align="left">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <code>elementsAllowedAriaLabel</code>
+      </td>
+      <td align="left">
+        <pre lang=js><code>[
+  "audio",
+  "applet",
+  "canvas",
+  "dl",
+  "embed",
+  "iframe",
+  "input",
+  "label",
+  "meter",
+  "object",
+  "span",
```

Comment:
I don't think `span` should be allowed

Reply:
@straker Are you able to link to the specific section of the ARIA spec that disallows `aria-label` on `span`? I found https://www.w3.org/TR/html-aria/, but I can't find anything that disallows it.

Reply:
So there is no spec documented reason for this (although it's coming). You can see where we got most of the concept in this pr https://github.com/dequelabs/axe-core/pull/2764. As it stands though, it's also a mix of a best practice since no screen reader supports it (https://github.com/dequelabs/axe-core/pull/2712#issuecomment-759732976).